product_landing_page/index.css

```diff
@@ -52,6 +54,18 @@ body {
   width: 300px;
   height: 30px;
 }
+.nav__item:hover {
```

Comment:
To taka ogólna uwaga, widzę, że jest  sporo pseudoselektorów `:hover` dla np. `__item`/`__link`/`--toggle` itp. Wychodzi na to, że chcemy, by dla każdego itema efekt `hover` był taki sam, nie ma znaczenia, czy to np. toggle/logo czy nie. Dlatego nie ma po co robić tego w różnych miejscach, każdy z itemów ma klasę `nav__item`, więc dla każdego z nich to zadziała.

Przenieś wszystkie atrybuty, które ustawiamy w pseudoselektorach `:hover` dot. klas, które napisałem, w jedno miejsce do `.nav__item:hover` (z media-query włącznie).